### ISSUE TITLE
A warning occurs during building `TranscriptionInfo.cpp`

### ISSUE THREAD

**Issue by maks00x** [tdlib/telegram-bot-api#333]
A warning occurs during building.

```
[ 94%] Building CXX object td/CMakeFiles/tdcore.dir/td/telegram/TranscriptionInfo.cpp.o
/tmp/tmp.JpPmLMxWdW/telegram-bot-api/td/td/telegram/TranscriptionInfo.cpp: In member function ‘td::vector<td::Promise<> > td::TranscriptionInfo::on_final_transcription(td::string&&, td::int64)’:
/tmp/tmp.JpPmLMxWdW/telegram-bot-api/td/td/telegram/TranscriptionInfo.cpp:127:19: warning: moving a local object in a return statement prevents copy elision [-Wpessimizing-move]
  127 |   return std::move(promises);
      |          ~~~~~~~~~^~~~~~~~~~
/tmp/tmp.JpPmLMxWdW/telegram-bot-api/td/td/telegram/TranscriptionInfo.cpp:127:19: note: remove ‘std::move’ call
```

I use commands in the `README.md` file:

```sh
git clone --recursive https://github.com/tdlib/telegram-bot-api.git
cd telegram-bot-api
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=Release ..
cmake --build . --target install # Here it occurs
```

That still has return code 0.

Information about dependencies and the compiler of mine:

```
zlib1g      1:1.2.11.dfsg-2ubuntu1.5
zlib1g-dev  1:1.2.11.dfsg-2ubuntu1.5 
openssl     1.1.1f-1ubuntu2.16
libssl-dev  1.1.1f-1ubuntu2.16 
gperf       3.1-1build1
gcc         4:9.3.0-1ubuntu2
cmake       3.16.3-1ubuntu1
```

I use Linux Mint 20.3 Cinnamon x64.

**Comment by levlam** [comment]
You can ignore the warning for now. It is a very minor performance warning, not affecting Bot API at all.

**Comment by levlam** [comment]
The warning was fixed in 6.3.2.
